readme.md

```diff
@@ -128,6 +128,17 @@ const getData = async id => {
 };
 ```
 
+Here we flatten things out using [another style](https://github.com/justsml/escape-from-callback-mountain), eliminating hardcoded async expressions from inside (most of) your logical code (i.e. `await` above).
+
+```js
+const getResults = data => Promise.all([data, process(data)]
+
+const getData = id => Storage
+	.find(id)
+	.all().then(getResults)
+	.all().then(prepare);
+```
```

Comment:
Where is the `.all()` coming from? Note that this example to is to illustrate using vanilla Promises (No Bluebird).